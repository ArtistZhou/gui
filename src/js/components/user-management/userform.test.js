import React from 'react';
import renderer from 'react-test-renderer';
import UserForm from './userform';

describe('UserForm Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<UserForm />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
