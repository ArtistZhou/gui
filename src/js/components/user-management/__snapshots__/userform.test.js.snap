// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserForm Component renders correctly 1`] = `
<form
  className=""
>
  <div
    className="flexbox"
    style={
      Object {
        "height": "min-content",
        "justifyContent": "flex-end",
        "margin": "24px 0 -16px 0",
      }
    }
  >
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-contained Mui-disabled Mui-disabled"
      disabled={true}
      id="submit_button"
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={-1}
      type="button"
    >
      <span
        className="MuiButton-label"
      />
    </button>
  </div>
</form>
`;
